src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -848,6 +851,19 @@ protected void addTranslations()
 		addExtra(MSItems.CASSETTE_DANCE_STAB, "desc", "Catboss - Dance-Stab-Dance");
 		add(MSItems.CASSETTE_RETRO_BATTLE, "Cassette");
 		addExtra(MSItems.CASSETTE_RETRO_BATTLE, "desc", "SinFrog - Retro Battle Theme");
+		
+		add("message.machinations", "Your blood shall fuel our machinations.");
+		add("message.stir", "They stir in your subconscious.");
+		add("message.suffering", "Your suffering grants us strength.");
+		add("message.will", "Our will is your will.");
+		add("message.done", "It is done.");
+		add("message.conspiracies", "You are a tool. A tool through which we we exert our conspiracies.");
+		add("message.waiting", "The moment will come when you embrace the darkness, you will accept it with open arms.");
+		add("message.strife", "Pain fuels us.");
+		add("message.search", "Search deeper. Gaze into the abyss with eyes wide open");
+		add("message.blessings", "With this struggle, we grant you our blessings.");
+		add("message.seek", "Seek out and consume the light.");
+		add("message.shadow", "The shadows do not work for you. They are your fellow vassals in service to us");
```

Comment:
I'm guessing it was separated from the weapon to make it appropriate for other situations.
Gonna suggest that you still let the translation key have some sort of closer connection to its use since these are all a category of messages.

Dunno if that made sense, so to be more explicit, I suggest changing these translation keys from "message.*" to "message.horrorterror.*"

Reply:
Understood!

Reply:
Done